tests/providers/ftp/hooks/test_ftp.py

```diff
@@ -120,6 +120,19 @@ def test_retrieve_file_with_callback(self):
             ftp_hook.retrieve_file(self.path, _buffer, callback=func)
         self.conn_mock.retrbinary.assert_called_once_with('RETR path', func)
 
+    def test_connection_success(self):
+        with fh.FTPHook() as ftp_hook:
+            status, msg = ftp_hook.test_connection()
+            assert status is True
+            assert msg == 'Connection successfully tested'
+        
+    def test_connection_failure(self):
+        self.conn_mock = mock.MagicMock(name='conn_mock', side_effect=Exception('Test'))
+        fh.FTPHook.get_conn = self.conn_mock
+        with fh.FTPHook() as ftp_hook:
+            status, msg = ftp_hook.test_connection()
+            assert status is False
+            assert msg == 'Test'
 
```

Comment:
```suggestion
            assert msg == 'Test'

```

Reply:
`Suggestion cannot be identical to original text. `